src/components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { Color } from "../../theme/util";
+
+const RADIO_BUTTON_WIDTH = 24;
+
+export interface RadioButtonProps {
+  children: string | React.ReactNode;
+  className?: string;
+  disabled?: boolean;
+  checked: boolean;
```

Comment:
shouldn't the checked attribute be optional? 
Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#checked

```suggestion
  checked?: boolean;
```
Check the js bin I scribbled to demo a potential error (based on this https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_radio)

https://jsbin.com/kucuzovezu/edit?html,output

the "others" will always be checked (even if it's value is `false`)

Reply:
Like we talked about on slack, I tested it in Storybook to make sure. If we don't provide a checked value to the RadioButton component, that input can never be checked visually, so that is why I thought it was necessary. 

I see the error in the bin you provided and that is a problem, but I couldn't reproduce it in our setup?  If it can, then maybe it could be optional, but then I am not sure how to make the child component function without it, even if I set the default of checked to false. 

we don't have to merge this yet though, i'm open to ideas or commits